docs/site_navigation/redirects.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Redirects
+parent: Site Navigation
+nav_order: 3
+---
+
+# Redirects
+
+The URL redirects are managed in **My Site > Redirects**, both Creators and Developers have access.
```

Comment:
This is fine, just an fyi now we have Roles/Permissions if we're going to say who access, for this it's specifically anyone with the 'Create and edit site pages' permission. This is not something reserved for Developer access or Owner (superuser) access. It's on the standard admin access too

Reply:
Rewritten!

Reply:
Hey @scsx I can't see that there have been any further changes since my comment. I think this is fine but you have said 'Rewritten' so just wanted to check if you've forgotten to push changes, or you're rewriting and these changes are to come soon?